recipes/aws-kvs-pic/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, tools, CMake
+
+required_conan_version = ">=1.33.0"
+
+class awskvspicConan(ConanFile):
+    name = "aws-kvs-pic"
+    license = "Apache-2.0"
+    homepage = "https://github.com/awslabs/amazon-kinesis-video-streams-pic"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("Platform Independent Code for Amazon Kinesis Video Streams")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, "fPIC": True}
+    generators = "cmake"
+    topics = ("conan", "aws", "kvs", "stream")
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["BUILD_DEPENDENCIES"] = False
+            self._cmake.configure()
+        return self._cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
```

Comment:
Feedback on components is welcome. I'm still not confident on this area of conan.
Particularly about the best practices surrounding target names / file names.
This lib only provides a pkg config interface so I made sure the names of the files matches that. What should we do for `cmake_find_package`?